.rubocop_todo.yml

```diff
@@ -0,0 +1,100 @@
+inherit_from: .rubocop-todo.yml
```

Comment:
@FranDepascuali Check out this file: You have everything disabled. 
This file should only have the AllCops and 
```ruby
Documentation:
  Enabled: false
LineLength:
  Max: 99
```